e2e/src/pages/app.po.ts

```diff
@@ -67,6 +67,15 @@ export class AppPage {
     }, 10000, 'failed to load search results');
   }
 
+  async clickAction(key: string) {
+    const downLoadRef = element.all(by.id('mvDownloadBtn')).filter(function (elm) {
```

Comment:
not sure what this is? are you just trying to click the download button? does this not work
this.clickElement(by.id('mvDownloadBtn'))